wdl/transforms/new-base/src/main/scala/wdl/transforms/base/ast2wdlom/AstToCallElement.scala

```diff
@@ -30,8 +30,15 @@ object AstToCallElement {
 
     val callBodyValidation: ErrorOr[Option[CallBodyElement]] = ast.getAttributeAsOptional[CallBodyElement]("body").toValidated
 
-    (callableNameValidation, aliasValidation, afterValidation, callBodyValidation) mapN { (name, alias, after, body) =>
-      CallElement(name, alias, after, body)
+    val srcLocValidation : ErrorOr[Option[SourceFileLocation]] = ast.getSourceLine match {
```

Comment:
All possible cases in this `match` result in a `validNel`, so is a validation really necessary?

I don't see a problem with referencing a previously-defined `val srcLocValidation : Option[SourceFileLocation]` in the `mapN { ... }` section.